src/netex-convertor/sharedHelpers.ts

```diff
@@ -216,52 +212,46 @@ export const replaceIWBusCoNocCode = (nocCode: string): string => {
 
 export const getCoreData = (
     operators: Operator[],
-    matchingData: PointToPointTicket | PeriodTicket | SchemeOperatorTicket,
+    ticket: PointToPointTicket | PeriodTicket | SchemeOperatorTicket,
 ): CoreData => {
-    const pointToPoint: boolean = matchingData.type === 'single' || matchingData.type === 'return';
+    const pointToPoint: boolean = ticket.type === 'single' || ticket.type === 'return';
 
     if (pointToPoint) {
-        const pointToPointMatchingData: PointToPointTicket = matchingData as PointToPointTicket;
+        const pointToPointTicket: PointToPointTicket = ticket as PointToPointTicket;
 
         return {
             opIdNocFormat: `noc:${operators[0].opId}`,
-            nocCodeFormat: `noc:${pointToPointMatchingData.nocCode}`,
+            nocCodeFormat: `noc:${pointToPointTicket.nocCode}`,
             currentDate: new Date(Date.now()),
             website: getCleanWebsite(operators[0].website),
-            brandingId: `op:${pointToPointMatchingData.nocCode}@brand`,
-            operatorIdentifier: pointToPointMatchingData.nocCode,
+            brandingId: `op:${pointToPointTicket.nocCode}@brand`,
+            operatorIdentifier: pointToPointTicket.nocCode,
             baseOperatorInfo: [],
             placeholderGroupOfProductsName: '',
             ticketUserConcat: '',
-            operatorPublicNameLineNameFormat: `${operators[0].operatorPublicName} ${pointToPointMatchingData.lineName}`,
-            nocCodeLineNameFormat: `${pointToPointMatchingData.nocCode}_${pointToPointMatchingData.lineName}`,
-            lineIdName: `Line_${pointToPointMatchingData.lineName}`,
-            lineName: pointToPointMatchingData.lineName,
-            isGeoZone: false,
-            isMultiProduct: false,
-            isMultiOperator: false,
-            isPointToPoint: pointToPoint,
-            type: matchingData.type,
-            isSchemeOperator: false,
+            operatorPublicNameLineNameFormat: `${operators[0].operatorPublicName} ${pointToPointTicket.lineName}`,
+            nocCodeLineNameFormat: `${pointToPointTicket.nocCode}_${pointToPointTicket.lineName}`,
+            lineIdName: `Line_${pointToPointTicket.lineName}`,
+            lineName: pointToPointTicket.lineName,
+            operatorName: pointToPointTicket.operatorShortName,
+            ticketType: pointToPointTicket.type,
         };
     }
-    const periodMatchingData: PeriodTicket | SchemeOperatorTicket = matchingData as PeriodTicket | SchemeOperatorTicket;
-    const baseOperatorInfo = isSchemeOperatorTicket(periodMatchingData)
-        ? getBaseSchemeOperatorInfo(periodMatchingData)
-        : operators.find(operator => operator.operatorPublicName === periodMatchingData.operatorName);
+    const periodTicket: PeriodTicket | SchemeOperatorTicket = ticket as PeriodTicket | SchemeOperatorTicket;
```

Comment:
When the type guards are in place we shouldn't need this variable anymore and we could just use ticket